libtock/hd44780.h

```diff
@@ -0,0 +1,47 @@
+/*
+	hd44780.h - C implementation of the LiquidCrystal class, providing basic
+	API for communicating with the 16x2 HD44780 LCD; 
+
+	The code was developed based on the basic Arduino Library for the LCD: 
+		https://github.com/arduino-libraries/LiquidCrystal.git
```

Comment:
Like #72, I believe the GPL derivation means we can't accept this :(

Reply:
That wasn't supposed to be here. Initially the library was in user-space, and when I modified it I forgot to remove the comment.